### PR TITLE
Update AppLifecycle StateNotifications.md

### PR DESCRIPTION
**Pull request by andreww-msft**

### CODE REVIEW

File: specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md [hunk 1]
```diff
@@ -239,147 +233,142 @@ The well-known
...
+| GUID_ACDC_POWER_SOURCE            | The system power source has changed.                         | `PowerSourceKind, PowerSourceKindChanged`                 |
+| GUID_BATTERY_PERCENTAGE_REMAINING | The remaining battery capacity has changed.                  | `RemainingChargePercent`, `RemainingChargePercentChanged` |
+| GUID_CONSOLE_DISPLAY_STATE        | The current monitor's display state has change. Apps should stop rendering graphics content when the monitor is off to reduce system power consumption. | _Not implemented_                                         |
+| GUID_GLOBAL_USER_PRESENCE         | The user status associated with any session has changed. This notification is only to services and other programs running in session 0. User-mode applications should register for GUID_SESSION_USER_PRESENCE instead. | _Not implemented_                                         |
+| GUID_IDLE_BACKGROUND_TASK         | The system is busy: now is a good time for apps to perform background or idle tasks that would otherwise prevent the computer from entering an idle state. | `SystemIdleStatus, SystemIdleStatusChanged`               |
```
Comment (huzaifa-d) [inline specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md:242]: There is no property for this one; just an event.

Comment (huzaifa-d) [reply specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md:242]: No 'SystemIdleStatus'

File: specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md [hunk 2]
```diff
@@ -239,147 +233,142 @@ The well-known
...
+| -------------------------------------------------------- | --------------------------------------- | ----------------------------------------------- |
+| PowerRegisterForEffectivePowerModeNotifications function | The effective power mode of the system. | `EffectivePowerMode, EffectivePowerModeChanged` |
 
 The existing PowerManager class exposes only static properties and events: all members are valid to
 be brought over to the Reunion version. Many existing enumerated values are brought over as-is as
 well.
 
 ```idl
-namespace Microsoft.Windows.System.Power
+namespace Microsoft.ProjectReunion
```
Comment (huzaifa-d) [inline specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md:261]: I think we might have to revert this once Jon is back. We will be moving forward with Windows.System.Power so we can keep this.

File: specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md [hunk 3]
```diff
@@ -239,147 +233,142 @@ The well-known
...
-    }
+    };
 
-    // Kind of the device's power source
-    enum PowerSourceKind {
-        AC,
-        DC,
-        ShortTerm
-    }
+    // Enums based on the Win32 PowerSettingRegisterNotification API.
```
Comment (huzaifa-d) [inline specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md:287]: Nice

File: specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md [hunk 4]
```diff
@@ -239,147 +233,142 @@ The well-known
...
+    enum EffectivePowerMode
+    {
+        BatterySaver,
+        BetterBattery,
+        Balanced,
         HighPerformance,
-        PowerSaver,
-        Balanced
-    }
+        MaxPerformance, // v1 last supported
```
Comment (huzaifa-d) [inline specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md:307]: I made these changes to my PR too - we should change the comment to something clearer like 
`    enum EffectivePowerMode
    {
        // EFFECTIVE_POWER_MODE_V1 values
        BatterySaver,
        BetterBattery,
        Balanced,
        HighPerformance,
        MaxPerformance,
        // EFFECTIVE_POWER_MODE_V2 additional values
        GameMode,
        MixedReality
    };`
    Open to something even better.

File: specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md (hunk 1 above)
Comment (andreww-msft) [reply specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md]: Fixed

File: specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md (hunk 2 above)
Comment (andreww-msft) [reply specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md]: Fixed

File: specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md (hunk 4 above)
Comment (andreww-msft) [reply specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md]: Fixed

**Review by huzaifa-d — APPROVED** [review]

**Pull request merged**